src/blosc2/lazyexpr.py

```diff
@@ -128,6 +177,30 @@ def save(self, **kwargs):
           if its source is a :ref:`SChunk`, :ref:`NDArray` or a :ref:`C2Array` (see :func:`blosc2.open` notes
           section for more info).
         * This is currently only supported for :ref:`LazyExpr`.
+
+        Examples
+        --------
+        >>> import blosc2
+        >>> import numpy as np
+        >>> dtype = np.float64
+        >>> shape = [3, 3]
+        >>> size = shape[0] * shape[1]
+        >>> a = np.linspace(0, 5, num=size, dtype=dtype).reshape(shape)
+        >>> b = np.linspace(0, 5, num=size, dtype=dtype).reshape(shape)
+        >>> # Define file paths for storing the arrays
+        >>> a_path = 'a_array.b2nd'
+        >>> b_path = 'b_array.b2nd'
+        >>> a1 = blosc2.asarray(a, urlpath=a_path, mode='w')
+        >>> b1 = blosc2.asarray(b, urlpath=b_path, mode='w')
+        >>> # Perform the mathematical operation to create a LazyExpr expression
+        >>> expr = a1 + b1
+        >>> # Save the LazyExpr to disk
+        >>> expr.save(urlpath='lazy_array.b2nd', mode='w')
+        >>> # Open and load the LazyExpr from disk
+        >>> disk_expr = blosc2.open('lazy_array.b2nd')
+        >>> f"The first two rows of the loaded LazyExpr data: {disk_expr[:2]}
```

Comment:
Too verbose.  Perhaps simply write `>>> disk_expr[:2]` ?  It should be explanatory enough IMO.